lstchain/tools/camera_calibration_param.json

```diff
@@ -34,11 +34,11 @@
        "r1_sample_start": 2,
       "r1_sample_end": 38
    },
-    "LocalPeakIntegrator":{
+    "LocalPeakWindowSum":{
      "window_shift": 5,
      "window_width":12
     },
```

Comment:
this values we may need to discuss/optimize. A window width of 12 corresponds to 12 ns, right? This is certainly too much for normal pulses, not sure if there is any requirement that needs to be fulfilled here, @moralejo, do you know?

Reply:
@ruben, this committed code is not what we are going to use, it is just an example. 
This are configuration files and the values can be change as one want. In the data of last days I am using 9 ns, but as you say one should do a study before fixing it